spec/support/db_helpers.rb

```diff
@@ -28,7 +28,7 @@ def connection
 
   def reset_database
     connection.execute('truncate table aggregates')
-    %w(events events_without_optimistic_locking).each do |table|
+    %w(events events_without_optimistic_locking).each do |_|
```

Comment:
We can actually remove this loop if we're not truncating both the tables.

I think this can be shortened to this too: `TRUNCATE TABLE table_name RESTART IDENTITY;`